WcaOnRails/app/inputs/date_picker_input.rb

```diff
@@ -3,10 +3,10 @@ def input(wrapper_options)
     set_html_options
     set_value_html_option
 
-    template.content_tag :div, class: 'input-group date datetimepicker' do
+    #template.content_tag :div, class: 'input-group date datetimepicker' do
       input = super(wrapper_options) # leave StringInput do the real rendering
-      input_button + input + utc_addon
-    end
+    #  input_button + input + utc_addon
```

Comment:
I forgot to actually delete the commented part. Actually I was waiting to see what you'd say about the thing (like the UTC part) :)

Let's try adding the utc_addon back...